README.md

```diff
@@ -1,29 +1,29 @@
-# Zendesk
+# Zendesk Support
 
-This package models Zendesk data from [Fivetran's connector](https://fivetran.com/docs/applications/zendesk). It uses data in the format described by [this ERD](https://docs.google.com/presentation/d/1AQv77L9WlDXqRS0gkdQTmg1HSUo-Znlcoq7CHg0JrP8).
+This package models Zendesk Support data from [Fivetran's connector](https://fivetran.com/docs/applications/zendesk). It uses data in the format described by [this ERD](https://docs.google.com/presentation/d/1AQv77L9WlDXqRS0gkdQTmg1HSUo-Znlcoq7CHg0JrP8).
 
-The main focus of this package is to enable you to better understand the performance of your Support team. Metrics focused on response times, resolution times and work time are calculated for you. 
+This package enables you to better understand the performance of your Support team. It calculates metrics focused on response times, resolution times, and work times for you to analyze. 
 
-#### Optional features (for Zendesk Professional or Enterprise users)
-- Metrics can be converted to business hours
-- SLA policy breaches are calculated
+### Optional features (for Zendesk Professional or Enterprise users)
+- Package converts metrics to business hours
+- Package calculates SLA policy breaches
```

Comment:
I wanted to use the active voice for the Optional Features section, but had to guess what the subject of the sentence should be. Please verify @fivetran-kristin 

Also, what do customers need to do to use these optional features?

Reply:
Instead of `Package` let's use `Models`  (multiple models are needed for this functionality)

I will update the README and outline what is needed to enable/disable this functionality.  Great catch!